### ISSUE TITLE
LibWeb: Crash on Acid3 in NavigableContainer::create_new_child_navigable()

### ISSUE THREAD

**Issue by awesomekling** [SerenityOS/serenity#21375]
Simplified repro, crashes immediately on load:

```html
<script>
let object = document.createElement('object');
object.setAttribute("data", "data:text/html,<script>document.write('<iframe>')<\/script>;");
</script>
```

**Comment by mpate132** [comment]
Can you share more information on this issue ?

**Comment by awesomekling** [comment]
No longer reproduces. Suspect this was fixed in 2f5ff14984d2ddfd33c2ce234167c48ddf9ac5f8
